cmd/lnmuxd/server.go

```diff
@@ -174,7 +174,9 @@ func (s *server) WaitForInvoiceSettled(ctx context.Context,
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
 
-	return &lnmuxrpc.WaitForInvoiceSettledResponse{}, nil
+	return &lnmuxrpc.WaitForInvoiceFinalEventResponse{
+		InvoiceStatus: statusToProto(invoiceStatus),
```

Comment:
Is this strict enough when only `SETTLED` and `FAILED` are returned? Alternatively it could be a boolean. But maybe the consistency with other calls is a plus too.